fcd/main.cpp

```diff
@@ -89,96 +91,18 @@ vector<T> parseListFlag(string flag, char sep) {
-//                           cl::Required, whitelist());
-// cl::list<unsigned long long> additionalEntryPoints(
-//     "other-entry",
-//     cl::desc(
-//         "Add entry point from virtual address (can be used multiple times)"),
-//     cl::CommaSeparated, whitelist());
-// cl::list<bool> partialDisassembly(
-//     "partial",
-//     cl::desc("Only decompile functions specified with --other-entry"),
-//     whitelist());
-// cl::list<bool> inputIsModule("module-in",
-//                              cl::desc("Input file is a LLVM module"),
-//                              whitelist());
-// cl::list<bool> outputIsModule("module-out", cl::desc("Output LLVM module"),
-//                               whitelist());
-
-// cl::list<string> additionalPasses(
-//     "opt",
-//     cl::desc("Insert LLVM optimization pass; a pass name ending in .py is "
-//              "interpreted as a Python script. Requires default pass
-//              pipeline."),
-//     whitelist());
-// cl::opt<string> customPassPipeline(
-//     "opt-pipeline", cl::desc("Customize pass pipeline. Empty string lets you
-//     "
-//                              "order passes through $EDITOR; otherwise, must
-//                              be "
-//                              "a whitespace-separated list of passes."),
-//     cl::init("default"), whitelist());
-// cl::list<string> headers(
-//     "header", cl::desc("Path of a header file to parse for function "
-//                        "declarations. Can be specified multiple times"),
-//     whitelist());
-// cl::list<string> frameworks(
-//     "framework", cl::desc("Path of an Apple framework that fcd should use for
-//     "
-//                           "declarations. Can be specified multiple times"),
-//     whitelist());
-// cl::list<string> headerSearchPath(
-//     "I", cl::desc("Additional directory to search headers in. Can be
-//     specified "
-//                   "multiple times"),
-//     whitelist());
-
-// template <int (*)()>  // templated to ensure multiple instatiation of the
-// static
-//                       // variables
-//                       inline int optCount(const cl::list<bool>& list) {
-//   static int count = 0;
-//   static bool counted = false;
-//   if (!counted) {
-//     for (bool opt : list) {
-//       count += opt ? 1 : -1;
-//     }
-//     counted = true;
-//   }
-//   return count;
-// }
-
-// inline int partialOptCount() {
-//   return optCount<partialOptCount>(partialDisassembly);
-// }
-
-// inline int moduleInCount() { return optCount<moduleInCount>(inputIsModule); }
-
-// inline int moduleOutCount() { return
-// optCount<moduleOutCount>(outputIsModule); }
-
-// void pruneOptionList(StringMap<cl::Option*>& list) {
-//   for (auto& pair : list) {
-//     if (!whitelist::isWhitelisted(*pair.second)) {
-//       pair.second->setHiddenFlag(cl::ReallyHidden);
-//     }
-//   }
-// }
+vector<string> headerSearchPath;
```

Comment:
Use explicit std::, also, if these are globals that are only needed in this file, then make them static and put them in an anonymous namespace.

Reply:
This will be a part of a separate PR which will refactor main. I'm planning on doing this after reenabling the AST transformation passes.